lib/msal-common/src/client/BaseClient.ts

```diff
@@ -47,7 +49,10 @@ export abstract class BaseClient {
     // Default authority object
     public authority: Authority;
 
-    protected constructor(configuration: ClientConfiguration) {
+    // Performance telemetry client
+    protected performanceClient?: IPerformanceClient;
+
+    protected constructor(configuration: ClientConfiguration, performanceClient?: IPerformanceClient) {
```

Comment:
If `performanceClient` is optional, this can be marked as a non-major bump.

Reply:
@ellymakuba You should either make this a required parameter (and keep as a major bump for msal-common), or only have this be a minor version for msal-common. Given this is also used by msal-node, I'm inclined to have it be optional for now (until the perf client work for msal-node is done)